native/CMakeLists.txt

```diff
@@ -2389,6 +2460,17 @@ if(USE_MIDDLEWARE)
 endif()
 
 ######## manual
+
+if(NOT OPENHARMONY)# TODO:May be removed later
```

Comment:
network can still not work?

Reply:
network Implemented in ts。Still need to block c++ code

Reply:
Ok, then why add TODO?

Reply:
hased removed